pages/enterprise/setup.rst

```diff
@@ -173,3 +173,11 @@ The license automatically applies to all nodes in your cluster without the need
           The same license is also attached as a text file in case it is wrongly formatted in the email.
 
 .. image:: /images/enterprise-license-1.png
+
+
+License Verification
+====================
+
+Some Graylog licenses requires to check their validation on a regular base. This includes the free Enterprise license with a specific amount of traffic included. 
```

Comment:
```text
When using Graylog Enterprise with a license issued after December 2017, Graylog regularly checks in with a license service at ``https://api.graylog.com/report`` in order to verify compliance with the license conditions.

This also includes the free Graylog Enterprise license with 5 GB/day.

If your network environment requires Graylog to use a proxy server in order to communicate with the external services via HTTPS, you'll have to configure the proxy server in the the :ref:`configuration file of Graylog <http_config>`. 
```

Reply:
not every License include the lookup - only those that included the flag, to be able to provide licenses for environments that are not able to connect to our backoffice. 

The amount of traffic might be adjusted from sales without a notice to us that's why I did not want to include any amount/countings in the text - just that this happens.

Reply:
I agree with Jan here. There are too many variables in this and if we start to document all of them, it will raise more questions than answer questions.

I would add this: "The Graylog web interface will show you details of the license, but if you are still unclear about the requirements, please contact sales with your questions."